src/main/java/com/revature/domain/Person.java

```diff
@@ -24,19 +24,23 @@
 	@Column(name = "p_lastname", nullable = false)
 	private String lastName;
 	
-	public Person() { 
-		// only exists for the sake of JPA
-	}
 
-	public Person(String firstName, String lastName) {
+	@Column(name = "p_role", nullable = false)
+	private int personRole;
+	
+	public Person() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1186)